pages/docs/config/sources/community.mdx

```diff
@@ -72,19 +72,55 @@ This will give you something like the following:
     snowflake:query:source (id, parent) - Config for a snowflake query Source. Work with multiple tables and build custom queries for its properties.
     snowflake:table:source (id, parent, from, mapping, with, high-water-mark) - Config for a snowflake table Source. Construct Sources and Properties without writing SQL.
 
+This list is determined by the plugins that you have installed. [Learn more about Plugins](/docs/plugins).
+
 ## Configuring Your Source
 
-The `generate` command attempts to make a _reasonable_ guess at the values in the config file it generated (in the `config/sources` directory). There are ellipses (`"..."`) in the places in which it couldn't make a reasonable guess. In the example above, after generating a Postgres Source, all but `appId` and `options.table` were pre-populated.
+Each Source has a common set of options, while other options are specific to the _type_ of Source. Here are the common options:
 
-<Alert variant="primary">
-  Note that the <code>appId</code> setting is where you would use the ID from
-  the App you created (e.g. <code>data_warehouse</code>).
-</Alert>
+- `class`: (**required**) Should be set to `"source"`.
+- `id`: (**required**) The unique value that defines this Source.
+- `name`: (**required**) A display name for the Source. Will be set to the `id` by default.
+- `type`: (**required**) The type of Source will vary based on the generator you're using. You likely don't want to change this value.
+- `appId`: (**required**) The `id` of the App (connection) that the Source uses.
+
+The generator makes a _reasonable_ guess at the values in the config file it generated (in the `config/sources` directory). There are ellipses (`"..."`) in the places in which it couldn't make a reasonable guess. In the example above, after generating a Postgres Source, all but `appId` and `options.table` were pre-populated.
 
 Other sections may be commented out but require some action by you. The best practice when configuring an Source is to read the comments, keys, and values within the generated file and fill in the appropriate values for your Source.
 
+### Bootstrapped Properties
+
+The very first Source you create in Grouparoo requires a Bootstrapped Property. Think of this as the primary key for all Profiles. Source config files provide you the ability to define this Bootstrapped Property. It will look like a simplified version of [a Property](/docs/config/properties/community).
```

Comment:
... This Bootstrapped Property must be unique to all of your Profiles, and is commonly a User ID or email address.